File: prototype/ts/ResultCloudData.ts
import { ExtendedModel, model, prop } from 'mobx-keystone';

import Onto from './Onto';
import { ResultCloudData as IResultCloudData } from './types';


@model('onto/ResultCloudData')
class ResultCloudData extends ExtendedModel(Onto, {
    some: prop<string>(''),
    data: prop<string>(''),
    right: prop<string>(''),
    here: prop<string>(''),
}) implements IResultCloudData {};

export default ResultCloudData;

File: packages/csharp/onto/OnToSharp/src/Types/ResultCloudData.cs
using GraphQLCodeGen;


namespace Types {

    public class ResultCloudData : Schema.ResultCloudData {
        public string data { get; set; }
        public string here { get; set; }
        public string right { get; set; }
        public string some { get; set; }

        public ResultCloudData(string some = "", string data = "", string right = "", string here = "") {
            this.some = some;
            this.data = data;
            this.right = right;
            this.here = here;
        }

        public override string ToString() => $"{some} {data} {right} {here}";
    }

}
